swift/Tooling/Templates/Screen (View Controller).xctemplate/___FILEBASENAME___Screen.swift

```diff
@@ -9,13 +9,17 @@ struct ___VARIABLE_productName___Screen: Screen {
 
     // It should also contain callbacks for any UI events, for example:
     // var onButtonTapped: () -> Void
+
+    var viewControllerDescription: ViewControllerDescription {
+        return ___VARIABLE_productName___ViewController.description(for: self)
```

Comment:
What's `___VARIABLE_productName___ViewController`?

Reply:
This is for the Xcode templates

Reply:
Oh cool!